x-pack/legacy/plugins/security/common/model/builtin_es_privileges.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface BuiltinESPrivileges {
```

Comment:
I'm not married to the name, but I thought I'd keep it consistent with what ES chose to call this.